packages/web/src/shared/utils/string.ts

```diff
@@ -0,0 +1,4 @@
+export function removeAfterHyphen(str: string): string {
+  const parts = str.split('-')
+  return parts[0].trim()
+}
```

Comment:
sugestão: definir isso como Regex para deixar esse função mais resiliente (replaceAll com `/-$/`, se for cortar o resto que está na frente do hífen, precisa de um wildcard antes do `$`)

Reply:
Isso só é usado no Leaderboard. Atualmente o mesmo quebraria sem essa função, mas assim que fizermos o Leaderboard usando nosso banco essa função vai ser inutilizada.